mx_nuget.package/MendixBinaries.nuspec

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>Mendix.LibGit2Sharp.NativeBinaries</id>
+    <version>0.1</version>
+    <authors>LibGit2Sharp contributors</authors>
+    <owners>mendix</owners>
+    <licenseUrl>https://raw.githubusercontent.com/libgit2/libgit2/master/COPYING</licenseUrl>
+    <projectUrl>https://github.com/libgit2/libgit2sharp.nativebinaries</projectUrl>
+    <repository url="https://github.com/mendix/libgit2sharp.nativebinaries.git" type="git"/>
+    <iconUrl>https://raw.githubusercontent.com/libgit2/libgit2sharp/master/square-logo.png</iconUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Native binaries for LibGit2Sharp</description>
+  </metadata>
+  <files>
+    <file src="../nuget.package/build/LibGit2Sharp.NativeBinaries.props" target="build/Mendix.LibGit2Sharp.NativeBinaries.props" />
+	<file src="./buildMultiTargeting/Mendix.LibGit2Sharp.NativeBinaries.props" />
```

Comment:
Are you sure this file will be put in the right directory without the `target` property? Maybe you just need to keep the previous way of specifying it (`buildMultiTargeting/**/*.*`).